hw/dv/data/tests/shadow_reg_errors_tests.hjson

```diff
@@ -0,0 +1,13 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  tests: [
```

Comment:
Add comment explaining that this should be imported by all IPs containing shadowed CSRs. Likewise for the testplan.